### ISSUE TITLE
Request to disable branch protection for branch eggvoice to clean up branch history

### ISSUE THREAD

**Issue by eggvoice** [earthlab-education/Earth-Analytics-2023-01-Intro#136]
TLDR: Request to disable branch protection for branch `eggvoice` to clean up branch history

I have been merging commits from the main branch into my personal branch through: 

1. `git merge` and `git push`
2. Pull request from main into personal branch

Number 1 seems to be much better, while Number 2 is creating an issue where my branch is ahead of main branch. 

Number 2 also creates a problem where any further updates are rejected due to the following issue:
`Updates were rejected because the tip of your current branch is behind its remote counterpart.`

I need the branch protection be temporarily disabled to clean up the branch commit history. Otherwise, I think I will have to create a whole new branch instead.

Any specific suggestions? Thank you!

**Comment by eculler** [comment]
Hi @eggvoice , you're branch will periodically get updates from main when I merge your work. If you open up your branch on GitHub, you should be able to create and merge a pull request. You have complete control over your branch, so I'm not sure what cleaning up you would be able to do with permissions on the main branch.

From the error message you are getting, it sounds like you are working on this outside of GitHub, like a Codespace? The message says that your current branch (eggvoice or main?) is behind its remote counterpart. That doesn't mean that eggvoice is behind main as you say; it means that the eggvoice you're working on is behind the eggvoice on GitHub.

Anyway I think I would need to know more about what your setup is to help with this (or we could take a look at office hours). Where are you running these commands? What does git status say? Are you able to pull successfully, or are you getting an error?

**Comment by eggvoice** [comment]
Closing this as discussed. I will be working within GitHub in the future for these changes. Thank you so much for your time!
